ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/admin/ApiPartyManagementService.scala

```diff
@@ -90,8 +90,12 @@ class ApiPartyManagementService private (
       .flatMap {
         case PartyAllocationResult.Ok(details) =>
           Future.successful(AllocatePartyResponse(Some(mapPartyDetails(details))))
+        case r @ PartyAllocationResult.Overloaded =>
```

Comment:
```suggestion
        case PartyAllocationResult.Overloaded =>
```
Looks unused as well.